mitype/commandline.py

```diff
@@ -155,3 +170,38 @@ def load_based_on_difficulty(difficulty_level=random.randrange(1, 6)):
 
     print("Select a difficulty level in range [1,5]")
     sys.exit(2)
+
+
+def show_history(N):
+    N = int(N)
+    history_file = 'mitype_history.csv'
+    history_path = os.path.join(os.path.expanduser('~'), history_file)
+    try:
+        with open(history_path, 'r') as file:
+            history_reader = csv.reader(file)
+            next(history_reader)
+
+            data = list(history_reader)
+            no_of_records = len(data)
+
+            if N >= len(data) or N == -1:
+                print(f"\nLast {no_of_records} records: ")
```

Comment:
This format is not supported in python 2.
Also, let's not print a \n at the beginning.